common/PersistenceFormat.js

```diff
@@ -10,32 +10,42 @@ var Q = require('q');
 
 module.exports = {
 
-    getEntryFullPath: function (entryId) {
+    getEntryBasePath: function (entryId) {
         return path.join(config.get('rootFolderPath'), entryId);
     },
 
-    getEntryRelativePath: function (entryId) {
-        return entryId;
+    getBasePathFromFull: function (path) {
+        let subString = path.match(/([01]_\w+\/[0-9]+\/)/)[1];
```

Comment:
maybe it is better to keep precompiled regex object  /([01]_\w+\/[0-9]+\/)/ instead